server/entity/damage/source.go

```diff
@@ -49,7 +49,9 @@ type (
 	SourceWitherEffect struct{}
 
 	// SourceFire is used for damage caused by being in fire.
-	SourceFire struct{}
+	SourceFire struct {
+		Campfire bool
```

Comment:
missing a doc here and maybe capitalize the f for consistency

Reply:
done 😄